packages/cubejs-backend-shared/src/promises.ts

```diff
@@ -273,9 +272,7 @@ export const asyncDebounce = <Ret, Arguments>(
   const cache = new Map<string, Promise<Ret>>();
 
   return async (...args: Arguments[]) => {
-    const key = crypto.createHash('md5')
-      .update(args.map((v) => JSON.stringify(v)).join(','))
-      .digest('hex');
+    const key = xxh3.xxh64(args.map((v) => JSON.stringify(v)).join(',')).toString(16);
```

Comment:
Let's make a function somewhere in `@cubejs-backend-shared,` with a name like `defaultHasherHex16` (or `defaultHasher` and a tiny interface over it) and use it everywhere in the code instead of defining and requiring `@node-rs/xxhash` in each package. 

This approach is how the Rust language does it: `DefaultHasher`. It will allow us to change it on the fly for testing via envs or replace the implementation whenever necessary.

Reply:
Absolutely agree! Great point!